components/event_dispatcher.rst

```diff
@@ -391,6 +393,10 @@ Take the following example of a subscriber that subscribes to the
                     ['onKernelResponsePost', -10],
                 ],
                 OrderPlacedEvent::NAME => 'onStoreOrder',
+                // you can also subscribe this way if you pass only
```

Comment:
```suggestion

                // you can also subscribe this way if you pass only
```

Reply:
Could you please add a version added directive below this code example? Thanks

Reply:
I'll do this @OskarStark , thx for the comments

Reply:
in fact, the feature was added in v4.3, thus in v4.4 (oldest branch maintained) it is always present, should I still add a version added 4.4 or 4.3 ?

Reply:
With 4.3 please, thanks